cts-java/src/test/java/org/ga4gh/cts/api/references/ReferenceSetsSearchIT.java

```diff
@@ -0,0 +1,149 @@
+package org.ga4gh.cts.api.references;
+
+import junitparams.JUnitParamsRunner;
+import org.apache.avro.AvroRemoteException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.TestData;
+import org.ga4gh.cts.api.reads.ReadsTests;
+import org.ga4gh.methods.GAException;
+import org.ga4gh.methods.SearchReferenceSetsRequest;
+import org.ga4gh.methods.SearchReferenceSetsResponse;
+import org.ga4gh.models.ReferenceSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.ga4gh.cts.api.Utils.aSingle;
+
+/**
+ * Reference sets-related compliance tests.
+ *
+ * @author Maciek Smuga-Otto
+ */
+@Category(ReferencesTests.class)
+@RunWith(JUnitParamsRunner.class)
+public class ReferenceSetsSearchIT {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Fetch reference sets and make sure we get some.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void checkSearchingReferenceSetsReturnsSome() throws AvroRemoteException {
+        final SearchReferenceSetsRequest req =
+                SearchReferenceSetsRequest.newBuilder().build();
+        final SearchReferenceSetsResponse resp = client.references.searchReferenceSets(req);
+
+        final List<ReferenceSet> sets = resp.getReferenceSets();
+        assertThat(sets).isNotEmpty();
+    }
+
+
+    /**
+     * Fetch reference sets and make sure they're well-formed.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void checkSearchingReferenceSetsReturnsWellFormed() throws AvroRemoteException {
+        final SearchReferenceSetsRequest req =
+                SearchReferenceSetsRequest.newBuilder().build();
+        final SearchReferenceSetsResponse resp = client.references.searchReferenceSets(req);
+
+        final List<ReferenceSet> sets = resp.getReferenceSets();
+
+        sets.stream().forEach(rs -> assertThat(rs.getId()).isNotNull());
+        sets.stream().forEach(rs -> assertThat(rs.getMd5checksum()).isNotNull());
```

Comment:
I had a test like that previously, and class Utils already includes a method called `looksLikeValidMd5`.  I can add a call to that, but I think it would be better to add a check that the MD5 value is correct.  It's a much stronger test.